packages/adblocker/test/parsing.test.ts

```diff
@@ -902,6 +909,30 @@ describe('Network filters', () => {
       });
     });
 
+    describe('removeparam', () => {
+      it('parses ~removeparam', () => {
+        // ~removeparam is not a valid option
+        network('||foo.com$~removeparam=utm', null);
+        network('||fo.com$~removeparam', null);
```

Comment:
How about `removeparam=~param` which seems to be supported by AdGuard (and referenced from uBO; unclear if uBO supports that). Let's make sure we reject these filters if we don't support them.

Reply:
Those tests are responsible for that. uBo marks `~removeparam` as an invalid option.

Reply:
Continuing in https://github.com/ghostery/adblocker/pull/4528#discussion_r1947942150